companion/src/modeledit/telemetry.h

```diff
@@ -25,6 +25,7 @@
 #include "eeprominterface.h"
 
 class AutoComboBox;
+class TimerEdit;
 
 namespace Ui {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this namespace to match the regular expression: [a-z][a-z_0-9]*+ [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3ANamespaceName)